kitsune/wiki/views.py

```diff
@@ -125,6 +125,32 @@ def document(request, document_slug, template=None, document=None):
             # and OK to fall back to parent (parent is approved).
             fallback_reason = 'no_translation'
 
+    # If any local has fallback locals and the docmunet is missing localization in that local
+    # So show the fallback local's document. There can be highest two fallback local for any local
+    # Fallback locals can be add in kitsune/wiki/config.py
+    if fallback_reason == 'no_translation' and (request.LANGUAGE_CODE in FALLBACK_LOCALS.keys()):
+        # Get fallback locals for the requested local
+        fallback_local_code = FALLBACK_LOCALS.get(request.LANGUAGE_CODE)
+        # Get the locals in which the document is already translated
+        translated_locales = doc.translations.all().values_list('locale', flat=True)
+        # If the document is translated in first fallback local, show that local version
+        if fallback_local_code[0] in translated_locales:
+            translation = doc.translated_to(fallback_local_code[0])
+            doc = Document.objects.get(locale=fallback_local_code[0],
+                                       slug=translation.slug)
+        else:
+            # If the document is missing translation in first fallback local
+            # try the second fallback local
+            if fallback_local_code[1] in translated_locales:
+                translation = doc.translated_to(fallback_local_code[1])
+                doc = Document.objects.get(locale=fallback_local_code[1],
+                                           slug=translation.slug)
+            # If the document is also missing translation in second fallback local
+            # Lets show the English version
+            else:
+                doc = get_object_or_404(Document, locale=settings.WIKI_DEFAULT_LANGUAGE,
+                                        slug=document_slug)
```

Comment:
I would write this a loop so that there can be any number of fallback languages. Something like

```python
translated_locales = doc.translation.all().values_list('locale', flat=True)
for fallback_locale in fallback_locale_codes:
    if fallback_locale in translated_locales:
        doc = doc.translated_to(fallback_locale)
        break
else:
    doc = get_object_or_404(Document, locale=settings.WIKI_DEFAULT_LANGAUGE, slug=document_slug)
```